pkg/util/containers/filter.go

```diff
@@ -364,24 +366,29 @@ func (cf Filter) IsExcluded(containerName, containerImage, podNamespace string)
 		}
 	}
 
-	return false
+	return cf.isExcludedByAnnotation(annotations, containerName)
```

Comment:
I think we might want to move this up further in the function, mainly because i think we want annotations to be used to override the env vars, but also because the annotation check should be faster than the env checks would be, so if people are using the annotation version they would then see improved performance (or maybe not, it is so slight anyway)

Reply:
Good catch ! It should be fixed by now.